.github/workflows/source.yml

```diff
@@ -2,8 +2,8 @@ name: source
 
 on:
   push:
-    tags:
-      - '**'
+    tags-ignore:
+      - '**-entrypoint'
```

Comment:
Shouldn't we do the same for packages? I checked `centos_8.yml` and it doesn't contain the exception.

Reply:
I see now, it is skipped in `.github/actions/pack-and-deploy/action.yml`.

Reply:
We have such an exception here https://github.com/tarantool/tarantool/blob/master/.github/actions/pack-and-deploy/action.yml#L24, but your variant looks better. There is no need to run testing on an `*-entrypoint` tag.

Reply:
Anyway, this is not a problem of this patchset, so I'll resolve the comment.